modules/ibi-extra-partition-ibi-install-iso.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="ibi-extra-partition-ibi-install-iso_{context}"]
+= Configuring additional partitions on the target host
+
+The installation ISO creates a partition for the `/var/lib/containers` directory as part of the image-based installation process. 
+
+You can create additional partitions by using the `coreosInstallerArgs` specification. For example, in hard disks with adequate storage, you might need an additional partition for storage options, such as {lvms-first}. 
+
+[NOTE]
+====
+The `var/lib/containers` partition requires at least 500 GB to ensure adequate disk space for precached images.
+====
+
+.Procedure
+
+. Edit the `image-based-installation-config.yaml` to configure additional partitions:
++
+.Example `image-based-installation-config.yaml` file
+[source,yaml]
+----
+skipDiskCleanup: true <1>
+coreosInstallerArgs: 
+   - "--save-partindex" <2>
+   - "6" <3>
+ignitionConfigOverride: |
+  {
+    "ignition": {
+      "version": "3.2.0" 
+    },
+    "storage": {
+      "disks": [
+        {
+          "device": "/dev/vda", <4>
```

Comment:
Maybe we should point that we are talking about installation disk?

Reply:
Thanks, I updated the callout to explain that. I also added the required specs at the start of the example YAML, which includes `installationDisk`. Perhaps it's more obvious when the `device` field == `installationDisk`

Reply:
let's use /dev/sda maybe?  /dev/vda implies virtual machine.